Makefile

```diff
@@ -14,25 +14,38 @@ TAG:=$(USER)$(shell date +%y%m%d_%H%M%S)
 
 DOCKER_TAG = $(DOCKER_PREFIX)$(IMAGE):$(TAG)
 
+CERT_TEMP := ./cert-tmp/
+
 # go test ./... and others run in vendor/ and cause problems (!)
-PACKAGES:=$(shell find . -type d -print | egrep -v "/(\.|vendor|static|templates|release|docs|json)")
+PACKAGES:=$(shell find . -type d -print | egrep -v "/(\.|vendor|static|templates|release|docs|json|cert-tmp)")
 #PACKAGES:=$(shell go list ./... | grep -v vendor)
 
 # Marker for whether vendor submodule is here or not already
 GRPC_DIR:=./vendor/google.golang.org/grpc
 
+# Run dependencies
+dependencies: submodule certs
+
 # Local targets:
-install: submodule
+install: dependencies
 	go install $(PACKAGES)
 
+# Generate certs for unit and release tests.
+certs:
```

Comment:
can you make this a dependency of the files or at least 1 of the file, eg $(CERT_TEMP)/server.cert

so:
```Makefile
certs: $(CERT_TEMP)/server.cert

$(CERT_TEMP)/server.cert: cert-gen
	./cert-gen
```





so make dependencies the 2nd time doesn't do anything ?

Reply:
ack